### DISCUSSION TITLE
The Incredible Hulk (2008) patch

### DISCUSSION THREAD

**Discussion by AD010** [xenia-canary/game-patches#60] [Requests]
please, create patch for The Incredible Hulk (2008), game have Title Update (TU) patch

**Comment by Margen67** [comment]
What sort of patch?

**Reply by MaxM2099** [reply]
I know this is old but he may be referring to the blur issue with the game where you can only select 1x scale or it will be a pixelated mess.

**Reply by mcluchko** [reply]
Yeah, we have a problem with scale resolution, because I can only select 1x scale (720p) or as MaxM2099 said it will be a pixelated mess
